.gitignore

```diff
@@ -1,3 +1,7 @@
+<<<<<<< HEAD
 .DS_Store/
 node_modules/
 dist/
+=======
+.DS_Store
+>>>>>>> main
```

Comment:
* The pull request doesn't contain the code of the 9th sprint: there is no `Api.js` and api integration. Please, fix the merge `conflicts`


`<<<<<<< HEAD` means that there is a **conflict** while merging a branch